src/util.ts

```diff
@@ -253,7 +253,19 @@ export async function touchTmpFile(path: string): Promise<() => void> {
   return callback;
 }
 
+export function pathEquals(path1: string, path2: string): boolean {
+  if (process.platform !== 'linux') {
+    path1 = path1.toLowerCase();
+    path2 = path2.toLowerCase();
+  }
+  return path1 === path2;
+}
+
 export function pathStartsWith(path1: string, path2: string): boolean {
+  if (process.platform !== 'linux') {
+    path1 = path1.toLowerCase();
+    path2 = path2.toLowerCase();
+  }
   const path1n = upath.normalize(path1).replace(/\/?$/, '/');
   const path2n = upath.normalize(path2).replace(/\/?$/, '/');
   return path1n.startsWith(path2n);
```

Comment:
For the same reason as above;

```suggestion
  const path1n = upath.normalize(path1).replace(/\/?$/, '/');
  const path2n = upath.normalize(path2).replace(/\/?$/, '/');
  return !path2n.relative(path1n).startsWith('..');
```

Reply:
normalize is not necessary if path.relative() is used, so the the code will be:

```suggestion
  return !upath.relative(path2, path1).startsWith('..');
```

I tested this in Windows. It works.